homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/style.css

```diff
@@ -6,7 +6,7 @@
   width: 575px;
   height: 479px;
   top: 140px;
-  left: 653px;
+  margin-left: 53px;
```

Comment:
please move this file to styles folder

Reply:
Please do not fix both `width` and `height` simultaneously for the non-fixed content block. It can break your layout
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/cf7f9e91-e0c2-44f0-9cb8-e7070a7adfd6)

Reply:
Please remove  the gaps 
<img width="1393" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/4f388127-cd6b-4ce6-a8b3-0258d3ad1910">

Reply:
fixed in 5015c401e868e8132d82c925762abaffb9ef6e48

Reply:
Please remove fixing the `height` of the block, which content could be changed. Here I've added some paragraphs and it has broken your layout
Fix in the next task
<img width="1656" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/b7c6f103-7996-4e26-810e-b1c4a5a80aa0">